index.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+
```

Comment:
delete empty lines?

Reply:
Sure. All files seem to start with 2 empty lines, though. I'll delete them all.

Reply:
I think that's the result of removing "use strict" by eslint fixer. Yep, lets remove them too.